src/libraries/System.Net.Http/src/System/Net/Http/GlobalHttpSettings.cs

```diff
@@ -77,5 +77,25 @@ private static double GetHttp2StreamWindowScaleThresholdMultiplier()
             }
         }
 #endif
+
+        internal static class HttpHandlerDefaults
+        {
+            public static int MaxConnectionsPerServer { get; } = GetMaxConnectionsPerServer();
+
+            private static int GetMaxConnectionsPerServer()
+            {
+                int value = RuntimeSettingParser.QueryRuntimeSettingInt32(
+                    "System.Net.SocketsHttpHandler.MaxConnectionsPerServer",
```

Comment:
If this is named `SocketsHttpHandler` it shouldn't affect `WinHttpHandler`. Or it should be named differently, depending on what's the goal of the setting.

Reply:
Scratch that, I can't reasonably propagate the value to WinHttpHandler (it is built for .NET Framework as well and there are unfortunate differences which prevent including necessary .cs files).

Reply:
Is there a reason this is not part of `internal static class SocketsHttpHandler` on line 20? Especially now that you decided to scope it purely to sockets handler.

Reply:
No, I forgot to move it. Should be done now